SECP-service/src/main/resources/assets/app/js/app.js

```diff
@@ -38,6 +38,11 @@ angular.module('SECP', ['ngResource', 'ngRoute', 'ui.bootstrap', 'ui.date',
         controller: 'LoginController',
         css: 'css/form.css',
       })
+      .when('/authenticate', {
+        templateUrl: 'views/login/authenticate.html',
+        controller: 'AuthenticateController',
+        css: 'css/login.css'
```

Comment:
not sure if i should add the logged in tag at this point or not @mohameddahrouj or @tosinoni

Reply:
We don’t know how this is gonna play out yet. So don’t put the logged in tag

Reply:
agreed.